libafl/src/feedbacks/map.rs

```diff
@@ -352,6 +355,7 @@ where
     }
 
     #[allow(clippy::wrong_self_convention)]
+    #[allow(clippy::needless_range_loop)]
```

Comment:
Why not do what it wants?

Reply:
You are right. I will fix this.
We'll have less instructions in this for-loop